tests/Unit/CssInlinerTest.php

```diff
@@ -2113,7 +2117,7 @@ public function matchingSelectorWithPseudoComponentCssRuleDataProvider()
      */
     public function inlineCssKeepsRuleWithPseudoComponentInMatchingSelector($css)
     {
-        $subject = $this->buildDebugSubject('<html><p><a id="a" class="a" href="a">foo</a></p></html>');
+        $subject = $this->buildDebugSubject('<html><p><a id="a" class="a" href="a">foo</a><input></p></html>');
```

Comment:
I'd prefer to have the `input` tag explicitly self-closing, i.e., to change it from `<input>` to `<input/>`. And while we're at it, can we add the required attributes to the `input` attribut eso the HTML would make sense in the real world, e.g. `<input type="text"/>?